distribution/pom.xml

```diff
@@ -22,11 +22,6 @@
          <artifactId>infinispan-query</artifactId>
       </dependency>
 
-      <dependency>
-         <groupId>org.infinispan</groupId>
-         <artifactId>infinispan-client-hotrod</artifactId>
-      </dependency>
-
```

Comment:
Is this right?

Reply:
No.. odd. I will fix it.

Reply:
Oh, actually, yes it is correct. The cachestore-remote already includes this dependency.